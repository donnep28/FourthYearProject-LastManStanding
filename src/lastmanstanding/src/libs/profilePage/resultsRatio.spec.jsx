import React from 'react';
import {shallow} from 'enzyme';
import ResultsRatio from './resultsRatio';

describe('ResultsRatio', () => {
  beforeAll(() => {
    jest.resetAllMocks();
  });

  const props = {
    myInfo: [
      {
        "Sub": "ecbc56df-d81a-410d-b399-4421b55ced32",
        "Username": "philip",
        "email": "philip.donnelly28@mail.dcu.ie",
        "favouriteTeam": "manchesterunited",
        "leagueIDs": [
          "Plips League#283625",
          "Glenn's Last Man Standing#522084",
          "Ladyboys Inc#819148",
          "The Last man standing#984785",
        ],
        "losses": "5",
        "wins": "0",
      }
    ],
  };

  describe('Snapshot', () => {
    it('renders correctly', () => {
      const component = shallow(
          <ResultsRatio {...props}/>,
      );
      expect(component).toMatchSnapshot();
    });
  });
});