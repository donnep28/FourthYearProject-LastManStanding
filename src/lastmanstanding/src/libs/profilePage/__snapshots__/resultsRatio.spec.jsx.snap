// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsRatio Snapshot renders correctly 1`] = `
<Fragment>
  <WithStyles(ForwardRef(ListItem))>
    Wins: 
    0
  </WithStyles(ForwardRef(ListItem))>
  <WithStyles(ForwardRef(ListItem))>
    Losses: 
    5
  </WithStyles(ForwardRef(ListItem))>
</Fragment>
`;
